controllers/user.js

```diff
@@ -62,4 +66,21 @@ const fetchTemplate = async (req, res, next) => {
   }
 }
 
-module.exports = {login, CMS, fetchTemplate}
+const deleteImage = async (req, res, next) => {
+  try{
+    let template = await TemplateModel.find({user: req.verified.user._id})
+    if(template.length!==0){
+    let {images} = template[0]
+    images = images.filter((item, index) => index!==req.body.image)
```

Comment:
Add appropriate spaces between equality operators, objects and function arguments in all files.